README.md

```diff
@@ -68,12 +68,26 @@ This task will deploy the following services to an Azure resource group:
 - [Application insights](https://azure.microsoft.com/it-it/services/application-insights/)
 - [Log analytics](https://azure.microsoft.com/en-au/services/log-analytics/)
 
-Most of the services are created running Terraform (see `infrastructure/azure.tf`).
+Most services get provisioned by Terraform (see `infrastructure/azure.tf`).
 
-Some services are actually unsupported by Terraform (CosmosDB database and collections, Functions, API manager);
-these ones are created by NodeJS scripts (`infrastructure/tasks`) that call the
+Some services aren't yet supported by Terraform (CosmosDB database and collections, [Functions](https://github.com/terraform-providers/terraform-provider-azurerm/issues/131), API manager);
+these ones are created by NodeJS scripts (`infrastructure/tasks`) that provision the services through the
 [Azure Resource Manager APIs](https://github.com/Azure/azure-sdk-for-node).
 
+#### Storing Terraform state remotely
+
+By default, Terraform stores state locally in a file named `terraform.tfstate`. 
+If you want to [store Terraform state remotely](https://www.terraform.io/docs/state/remote.html)
```

Comment:
This is not a choice that you can easily change afterwards, we should make a decision and stick to it. We can either store the tfstate in a private git repo or in an Azure bucket, Azure is probably easier/safer, repo is more future proof.

Reply:
could be nice to make an ADR for this